src/validation.cpp

```diff
@@ -4754,9 +4754,7 @@ bool LoadExternalBlockFile(const CChainParams& chainparams, FILE* fileIn, FlatFi
     } catch (const std::runtime_error& e) {
         AbortNode(std::string("System error: ") + e.what());
     }
-    if (nLoaded > 0)
-        LogPrintf("Loaded %i blocks from external file in %dms\n", nLoaded, GetTimeMillis() - nStart);
-    return nLoaded > 0;
+    LogPrintf("Loaded %i blocks from external file in %dms\n", nLoaded, GetTimeMillis() - nStart);
```

Comment:
This change adds log entry if `nLoaded == 0`. Is it intentional?

Reply:
Yes. `LoadExternalBlockFile` is only called when files are actually read. If `nLoaded` is zero, something has gone wrong and we should be verbose about that.